### DISCUSSION TITLE
Request: installation via NPM

### DISCUSSION THREAD

**Discussion by shineholiday** [sveltia/sveltia-cms#42] [Ideas]
I hope this feature will be available soon.  

Like your project. Thank u so much.

**Comment by kyoshino** [comment]
Thanks for your input! My plan is to mimic the [behaviour of Netlify/Decap CMS](https://decapcms.org/docs/add-to-your-site/#installing-with-npm). Can’t provide an ETA at this time but hopefully it’s not so complicated 🙂

**Comment by kyoshino** [comment]
As of [v0.21.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.21.0), Sveltia CMS can be installed as an npm package 🚀
